app/build.gradle

```diff
@@ -64,12 +52,15 @@ dependencies {
     exclude group: 'android.arch.lifecycle'
     exclude group: 'com.google.android.gms'
   })
-  implementation dependenciesList.mapboxGeocoding
+  implementation (dependenciesList.mapboxGeocoding, {
+    exclude group: 'com.squareup.okhttp3'
+  })
```

Comment:
why do we exclude the okhttp specifically?

Reply:
It will conflict with another version of okhttp and cause a compile error.